apps/framework-cli/src/framework/python/wrappers/scripts/python_worker_wrapper/workflow.py

```
@@ -106,110 +97,57 @@ def _parse_task_timeout(self, timeout_str: Optional[str]) -> Optional[timedelta]
                 for child_task in task.config.on_complete:
                     child_result = await self._execute_activity_with_state(
                         wf,
                         child_task,
                         result.data if hasattr(result, "data") else result,
                     )
-                    if isinstance(child_result, ContinueAsNewData):
-                        return child_result  # Propagate continue-as-new
                     results.extend(child_result)
 
             return results
         except Exception as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)